src/main/java/com/microsoft/azure/vmagent/AzureVMAgentCleanUpTask.java

```diff
@@ -432,7 +439,7 @@ public Void call() throws Exception {
     }
 
     public AzureVMCloud getCloud(final String cloudName) {
-        return Jenkins.getInstance() == null ? null : (AzureVMCloud) Jenkins.getInstance().getCloud(cloudName);
+        return (AzureVMCloud) Jenkins.getInstance().getCloud(cloudName);
```

Comment:
OK, `@CheckFurNull `become `@Nonnull ` in 2.60-LTS, I forgot to change back here.